dvc/repo/pkg.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from dvc.output.local import OutputLOCAL
+
+
+class Pkg(object):
+    PKG_DIR = "pkg"
+
+    def __init__(self, repo):
+        self.repo = repo
+        self.pkg_dir = os.path.join(repo.dvc_dir, self.PKG_DIR)
+
+    def _collect(self, name=None):
+        if name:
+            from dvc.pkg import Pkg
+
+            return [Pkg.create(self.repo, name)]
+
+        pkgs = []
+        for stage in self.repo.stages():
+            for dep in stage.deps:
+                if dep.pkg:
+                    pkgs.append(dep.pkg)
+        return pkgs
+
+    def install(self, name=None):
+        for pkg in self._collect(name):
+            pkg.install()
+
+    def uninstall(self, name=None):
+        for pkg in self._collect(name):
+            pkg.uninstall()
+
+    def imp(self, pkg_name, pkg_path, out=None):
+        src = {
+            OutputLOCAL.PARAM_PATH: pkg_path,
+            OutputLOCAL.PARAM_PKG: pkg_name,
+        }
```

Comment:
why don't we just create an URL right here? instead of propagating this all the way down to OutputLOCAL (which is extremely confusing by itself like I mentioned in the output itself). If we decided to reuse import and import operated with URLs (and we actually have to convert it to URL anyway when we create a DVC-file), let's use URLs from the very beginning. It's very hard to follow all these manipulations with dicts, strings, etc.

Reply:
@shcheklein There are no urls no more. No `pkg+` or `pkg://mypkg` notation.